core/src/main/scala/org/apache/spark/util/AkkaUtils.scala

```diff
@@ -106,7 +104,6 @@ private[spark] object AkkaUtils extends Logging {
       |akka.remote.secure-cookie = "$secureCookie"
       |akka.remote.transport-failure-detector.heartbeat-interval = $akkaHeartBeatInterval s
       |akka.remote.transport-failure-detector.acceptable-heartbeat-pause = $akkaHeartBeatPauses s
-      |akka.remote.transport-failure-detector.threshold = $akkaFailureDetector
```

Comment:
Just checked looks like the implementation class for the Failure Detector is changed, and so it does not need threshold property. The most idiomatic solution is like I have already said - provide an empty implementation.